views/js/controller/runner/runner.js

```diff
@@ -115,7 +115,10 @@ define([
             var runnerOptions = _.defaults({}, options, defaults);
             var exitReason;
             var $container = $('.runner');
-            var logger     = loggerFactory('controller/runner', { runnerOptions : runnerOptions });
+            var logger = loggerFactory('controller/runner', {
+                serviceCallId : runnerOptions.serviceCallId,
+                plugins : Object.keys(runnerOptions.plugins)
```

Comment:
I would suggest that `plugins : Object.keys(runnerOptions.plugins)` could be moved to the initial `logger.debug()` call on L267, because we only want that information logged one time; and the current L267 is incorrect because it logs pluginFactories and not plugin names.

Reply:
With the suggested change, I've verified that it logs the desired strings in both `console` and `http` implementations.

Reply:
But in that case we will not see the plugins list in the log because we do not send debug level messages to the server.